src/Audio/babylon.audioEngine.ts

```diff
@@ -102,7 +142,8 @@
                     this._connectedAnalyser.stopDebugCanvas();
                     this._connectedAnalyser.dispose();
                     this.masterGain.disconnect();
-                    this.masterGain.connect(this._audioContext.destination);
+                    // Looks Strange to me.

```

Comment:
Agree. Perhaps it is use to restore default sound configuration?

Reply:
ohhh gotcha I ll double check during the tests